File: projects/m1/015-is-it-a-leap-year/solutions/Davide Di Giacomo/is_it_a_leap_year.py
print ("Insert a year to determinate if is leap or not!")
print ("Insert a year:")
year = int(input())

if year % 400 == 0:
    print (f"{year} is a leap year!")
    
elif year % 100 == 0:
    print (f"{year} isn't a leap year!")
    
elif year % 4 == 0:
    print (f"{year} is a leap year!")
    
else:
    print (f"{year} isn't a leap year!")

File: projects/m1/015-is-it-a-leap-year/solutions/stefandifede/is-it-a-leap-year.js
const yearInput = prompt("Insert a year")

function isItLeap() {
    if (yearInput % 400 == 0 || yearInput % 100 != 0 && yearInput % 4 == 0) {
        alert(yearInput + " is a leap year")
    } else {
        alert(yearInput + " is not a leap year" )
    }
}

isItLeap();